packages/types/src/indexed-db.ts

```diff
@@ -171,6 +178,10 @@ export interface PenumbraDb extends DBSchema {
     key: string; // bech32-encoded validator identity key
     value: Jsonified<ValidatorInfo>;
   };
+  PRICES: {
+    key: string[]; // composite key [base64 EstimatedPrice['priced_asset'][inner'], base64 EstimatedPrice['numeraire']['inner']]
```

Comment:
Using the composite key ensures that we will not have two records for the combination `priced_asset:numeraire` and will always store only the most recent value